plutus-conformance/plutus-conformance.cabal

```diff
@@ -0,0 +1,134 @@
+cabal-version: 3.0
+name: plutus-conformance
+version: 1.0.0.0
+license: Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+maintainer: marty.stumpf@iohk.io
+author: Plutus Core Team
+synopsis: Conformance Test Suite for Plutus Core
+description:
+    Comprehensive Conformance Test Suite for Plutus Core.
+category: Language, Plutus, Conformance
+build-type: Simple
+extra-doc-files: README.md
+extra-source-files:
```

Comment:
wrong, but should maybe include the test files? I think we can worry about that later